code-organization.md

```diff
@@ -237,3 +238,61 @@ But what it gives us is extreme consistency by the most correct thing
 also the easiest.
 
 For a summary, please scroll back up and see the TL;DR section.
+
+## Inheritance
+
+We use a functional approach to inheritance, in this style:
+```
+var animal = function(options) {
+  var self = {},
+      location = 0,
+      speed = 5;
+  self.name = options.name;
+  
+  self.run = function(time) {
+    self.location += time * speed;
```

Comment:
This sets `self.location`, which is initially `undefined`, so that causes `self.location` to be set to `NaN`. The `self.` part should be removed.

Similar situation below in `getLocation`